virttest/utils_test/__init__.py

```diff
@@ -2626,6 +2626,50 @@ def check_dest_vm_network(vm, vm_ip, remote_host, username, password,
                                   % (vm.name, results_stdout_52lts(result)))
 
 
+def check_remote_vm_disks(vm, params):
+    """
+    Check disks in remote vm are working well with I/O.
+
+    :param vm: VM object on remote host
+    :param params: the dict used for parameters
+
+    :raise: exceptions.TestFail if fail to write to the disk
+    """
+    cmd = "fdisk -l|grep '^Disk /dev'|cut -d: -f1|cut -d' ' -f2"
+    vm_ip = vm.get_address()
+    vm_pwd = params.get("password", "redhat")
+    mnt_dir = "/tmp/mnt_dir"
+    dd_cmd = "dd if=/dev/zero of=%s/test bs=1024 count=512 && sync" % mnt_dir
+    remote_vm_obj = RemoteVMManager(params)
+    remote_vm_obj.check_network(vm_ip)
+    remote_vm_obj.setup_ssh_auth(vm_ip, vm_pwd, timeout=60)
+    cmdres = remote_vm_obj.run_command(vm_ip, cmd)
+    disks = cmdres.stdout.strip().split("\n")
+    logging.debug("Get disks in remote VM: %s", disks)
+
+    for disk in disks:
+        if disk.count('/dev/vda') or disk.count('mapper'):
```

Comment:
need figure out one way to detect system disk

Reply:
Update

Reply:
This is not valid any more. The part was removed already.